docs/docsite/rst/network_getting_started.rst

```diff
@@ -0,0 +1,246 @@
+.. network-getting-started-example:
+
+*******************************
+Network getting started example
+*******************************
+
+.. contents:: Topics
+
+
+Overview
+========
+
+Objective
+---------
+
+I want to understand how Ansible can be used to connect to multiple network devices.
+
+.. FIXME FUTURE Gundalow - Link to examples index once created
+
+.. note:: Examples, not production code
+   This is not for production use, it's more to demonstrate the different areas that need to be considered and provide a foundation level knowledge.
+
+Prerequisites
+-------------
+
+* Ansible 2.3 (or higher) installed :doc:`intro_installation`
+* One or more network device compatible with Ansible
+* Basic understanding of YAML :doc:`YAMLSyntax`
+* Basic understanding of Jinja2 Templates :doc:`playbooks_templating`
+
+.. FIXME FUTURE Gundalow - Once created we will link to the connection table here (which platforms support network_cli & credentials through inventory)
+.. FIXME FUTURE Gundalow -  Using ``ansible_ssh_pass`` will not work for REST transports such as ``eapi``, ``nxapi`` - Once documented in above FIXME add details her
+
+Audience
+--------
+
+* Basic Linux (comfortable on the command line)
+* Basic Network switch & router configuration knowledge
+
+
+Solution
+=========
+
+Create an inventory file.
+
+
+**Inventory file**
+
+Create a file called ``inventory``, containing:
+
+.. code-block::
+
+   [switches:children]
+   ios
+   vyos
+
+   [ios]
+   ios01.example.net ansible_connection=local ansible_user=cisco ansible_ssh_pass=cisco
+
+   [vyos]
+   vyos01.example.net ansible_connection=local ansible_user=admin ansible_ssh_pass=admin
```

Comment:
Can we have small intro to variables host_vars and group_vars, along with inventory. ansible_connection can go into group_vars and credentials to host_vars.  We can leave this as such and have some other variables as well.

Reply:
What exactly do you think I should add?

Reply:
A simple generic comment statement in (group/all) like  provisioning_comment: "Configured by Ansible" or Syslog server IP etc.  Hostname/username/password in host_vars, would do.

Reply:
Thanks for the extra detail.

For context this `network_getting_started.rst` isn't meant to explain every part of Ansible. Along with setting hostname I could set set syslog server using a group_var.

We want to move people away from using `provider:` and passing authentication & host details in the task, hence this documentation using Ansible in a more "Ansible like way".